libs/map/src/lib/components/baselayer/vl-map-base-layer.ts

```diff
@@ -149,26 +152,48 @@ export class VlMapBaseLayer extends BaseElementOfType(HTMLElement) {
     }
 
     _createBaseLayer() {
+        const hasBackgroundLayer = this.hasAttribute('background-layer');
+
+        const createTileLayer = () =>
+            new TileLayer(<any>{
+                title: this.title,
+                type: 'base',
+                source: this._WMTSSource,
+            });
+
+        const createVectorLayer = () =>
+            new VectorLayer({
+                source: this._vectorSource,
+                style: new OlStyle({
+                    stroke: new OlStyleStroke({
+                        color: 'rgba(0, 0, 0, 1.0)',
+                        width: 1,
+                    }),
+                    fill: new OlStyleFill({
+                        color: 'rgba(255, 0, 0, 1.0)',
+                    }),
+                }),
+            });
+
+        const createCartoWebBackground = (layers: string, opacity: number) =>
+            new TileLayer({
+                source: new TileWMS({
+                    params: { FORMAT: 'image/png', LAYERS: layers },
+                    url: 'https://cartoweb.wms.ngi.be/service?version%3D1.3.0',
```

Comment:
Is dit versienummer nodig?

Reply:
zonder versie nummer werkt het ook

aangepast